style.css

```diff
@@ -0,0 +1,82 @@
+body {
+background-color: white;
+font-family: arial;
+}
+
+.img {
```

Comment:
adding padding-left to an image to center it can cause problems - as soon as you resize the screen.  can you think of another way to center the image?  hint: set the image width to a percentage and then look at your text-align options on this class